addons/payment_mercado_pago/README.md

```diff
@@ -1,41 +1,42 @@
 # Mercado Pago
 
-## Implementation details
+## Technical details
 
-### Supported features
+APIs:
+- [Checkout Pro](https://www.mercadopago.com.mx/developers/en/docs/checkout-pro/landing)
+- [Checkout API](https://www.mercadopago.com.mx/developers/en/docs/checkout-api/landing)
+
+This module integrates Mercado Pago using a combination of "Checkout Pro" and "Checkout API". The
+generic payment with redirection flow based on form submission provided by the `payment` module is
+used to initiate the payment with the same request payload as Checkout Pro's JavaScript SDK would.
+The remaining API calls are made to the Checkout API. It was not possible to only integrate with
+Checkout Pro as it only allows redirecting customers to the payment page, nor with only the Checkout
+API only as it requires building a custom payment form to accept direct payments from the merchant's
+website.
+
+## Supported features
 
 - Payment with redirection flow
-- Several payment methods such as credit cards, debit cards, and
-  [others](https://www.mercadopago.com.mx/developers/en/docs/checkout-api/payment-methods/other-payment-methods).
-- [Webhook](https://www.mercadopago.com.mx/developers/en/docs/notifications/webhooks/webhooks)
-  notifications.
+- Webhook notifications
 
-### Not implemented features
+## Not implemented features
 
-- [Manual capture](https://www.mercadopago.com.mx/developers/en/docs/checkout-api/payment-management/capture-authorized-payment).
-- [Partial refunds](https://www.mercadopago.com.mx/developers/en/docs/checkout-api/payment-management/cancellations-and-refunds).
+- [Manual capture](https://www.mercadopago.com.mx/developers/en/docs/checkout-api/payment-management/capture-authorized-payment)
+- [Full and partial refunds](https://www.mercadopago.com.mx/developers/en/docs/checkout-api/payment-management/cancellations-and-refunds)
 
-### API and gateway
+## Module history
 
-We choose to integrate with a combination of the
-[Checkout Pro](https://www.mercadopago.com.mx/developers/en/docs/checkout-pro/landing) and
-[Checkout API](https://www.mercadopago.com.mx/developers/en/docs/checkout-api/landing) solutions:
-The payment with redirection flow is initiated by sending a client HTTP request with a form-encoded
-payload like Checkout Pro's JavaScript SDK does under the hood. The remaining API calls are made
-according to the Checkout API's documentation. It was not possible to integrate with Checkout Pro
-only as it only allows redirecting customers to the payment page, nor with the Checkout API only as
-it requires building a custom payment form to accept direct payments from the merchant's website.
+- `16.0`
+  - The first version of the module is merged. odoo/odoo#83957
 
-The other gateways were ruled out. See the task's dev notes for the details on the other gateways.
+## Testing instructions
 
-The API implemented by this module is not versioned.
+https://www.mercadopago.com.mx/developers/en/docs/checkout-api/integration-test/test-cards
```

Comment:
Sorry, the information you want to view does nor exist or has been moved :(
`https://www.mercadopago.com.mx/developers/en/docs/checkout-api/additional-content/your-integrations/test/cards`